partner_prospect/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Partner Prospect",
+    "version": "13.0.1.0.0",
+    "category": "Sales Management",
```

Comment:
@sudhir-erpharbor Could you change this to 'Sales' as it won't appear in OCA shop in the good category?

Reply:
Done.